prescription/store/mongo/store_test.go

```diff
@@ -88,3 +67,16 @@ var _ = Describe("Store", Label("mongodb", "slow", "integration"), func() {
 		})
 	})
 })
+
+var suiteStore *prescriptionStoreMongo.PrescriptionStore
```

Comment:
I'd strongly recommend adding this "common" code in a common location like the `*_suite_test.go` file for each package. That way, if anyone wants to find the suite store then just need to look in one place.